### PR TITLE
🌐 fix(i18n): improve FR translations and correct typos

### PR DESCRIPTION
**Pull request by be-next**
## Summary

I have corrected and improved certain suggestions in the French (FR) translation file.

### Type of change

- [ ] Bug fix (fixes an issue without altering functionality)
- [ ] New feature (adds non-breaking functionality)
- [ ] Breaking change (alters existing functionality)
- [x] UI/UX improvement (enhances user interface without altering functionality)
- [ ] Refactor (improves code quality without altering functionality)
- [ ] Documentation update
- [ ] Other (please describe below)

---

## Checklist

- [ ] I have verified the accessibility of my changes
- [x] I have tested all possible scenarios for this change
- [ ] I have updated `theme.toml` with a sane default for the feature
- [ ] I have made corresponding changes to the documentation:
  - [ ] Updated `config.toml` comments
  - [ ] Updated `theme.toml` comments
  - [ ] Updated "Mastering tabi" post in English
  - [ ] (Optional) Updated "Mastering tabi" post in Spanish
  - [ ] (Optional) Updated "Mastering tabi" post in Catalan

### CODE REVIEW

File: i18n/fr.toml [hunk 1]
```diff
@@ -62,8 +58,8 @@ table_of_contents = "Table des matières"
 load_comments = "Afficher les commentaires"
 
 # Copy code block button.
-copied = "Copié!"
-copy_code_to_clipboard = "Copier le code dans le presse-papiers"
+copied = "Copié !"
```
Comment (welpo) [inline i18n/fr.toml:61]: Is the space before the exclamation mark intentional?

Comment (be-next) [reply i18n/fr.toml:61]: Yes, that's right. In French typography, the punctuation marks '!', '?' and ':' are always preceded by a non-breaking space.

Comment (welpo) [reply i18n/fr.toml:61]: Interesting!

**Comment by welpo** [comment]
Thank you so much for taking the time to fix these issues! 🫶

**Pull request merged**